tests/unit/OptionsListUtilsTest.ts

```diff
@@ -517,46 +514,34 @@ describe('OptionsListUtils', () => {
 
     it('getOptions() for group Chat', () => {
         // When we call getOptions() with no search value
-        let results = OptionsListUtils.getOptions({reports: OPTIONS.reports, personalDetails: OPTIONS.personalDetails});
+        let results = OptionsListUtils.getValidOptions({reports: OPTIONS.reports, personalDetails: OPTIONS.personalDetails});
 
         // We should expect all the personalDetails to show except the currently logged in user
         // Filtering of personalDetails that have reports is done in filterOptions
         expect(results.personalDetails.length).toBe(Object.values(OPTIONS.personalDetails).length - 1);
 
-        // All personal details including those that have reports should be returned
-        // We should expect personal details sorted alphabetically
-        expect(results.personalDetails.at(0)?.text).toBe('Black Panther');
-        expect(results.personalDetails.at(1)?.text).toBe('Black Widow');
-        expect(results.personalDetails.at(2)?.text).toBe('Captain America');
-        expect(results.personalDetails.at(3)?.text).toBe('Invisible Woman');
-        expect(results.personalDetails.at(4)?.text).toBe('Mister Fantastic');
-        expect(results.personalDetails.at(5)?.text).toBe('Mr Sinister');
-        expect(results.personalDetails.at(6)?.text).toBe('Spider-Man');
-        expect(results.personalDetails.at(7)?.text).toBe('The Incredible Hulk');
-        expect(results.personalDetails.at(8)?.text).toBe('Thor');
-
         // And none of our personalDetails should include any of the users with recent reports
         const reportLogins = results.recentReports.map((reportOption) => reportOption.login);
         const personalDetailsOverlapWithReports = results.personalDetails.every((personalDetailOption) => reportLogins.includes(personalDetailOption.login));
         expect(personalDetailsOverlapWithReports).toBe(false);
 
-        // When we provide no selected options to getOptions()
-        results = OptionsListUtils.getOptions({reports: OPTIONS.reports, personalDetails: OPTIONS.personalDetails}, {maxRecentReportsToShow: 5});
+        // When we limit getOptions()
+        results = OptionsListUtils.getValidOptions({reports: OPTIONS.reports, personalDetails: OPTIONS.personalDetails});
 
         // Then one of our older report options (not in our five most recent) should appear in the personalDetails
         // but not in recentReports
-        expect(results.recentReports.every((option) => option.login !== 'peterparker@expensify.com')).toBe(true);
-        expect(results.personalDetails.every((option) => option.login !== 'peterparker@expensify.com')).toBe(false);
+        // expect(results.recentReports.every((option) => option.login !== 'peterparker@expensify.com')).toBe(true);
+        // expect(results.personalDetails.every((option) => option.login !== 'peterparker@expensify.com')).toBe(false);
```

Comment:
Why are these commented out? Please uncomment or remove and explain why.

Reply:
I removed this now (thanks for pointing it out!). This test case only worked when limiting the options using `maxRecentReportsToShow` (because it changes the results in the options).

With these changes `getValidOptions` doesn't offer the possibility to anymore to limit the results (as no call side in the app needs it, we only need it when filtering and ordering)